CMakeLists.txt

```diff
@@ -354,7 +355,9 @@ endif()
 install(DIRECTORY include/tbb DESTINATION ${TBB_INSTALL_INCLUDE_DIR})
 
 # version_string.ver
-if (UNIX)
+if (TBB_NO_DATE)
```

Comment:
Could you merge the TBB_NO_CASE branch with the one at the bottom (which is also "Unknown")?